api/api.proto

```diff
@@ -2014,10 +2014,14 @@ message SdkCloudBackupCreateRequest {
   string credential_id = 2;
   // Full indicates if full backup is desired even though incremental is possible
   bool full = 3;
+  // TaskId is the optional unique id of the task performing this backup
+  string task_id = 4;
 }
 
 // Empty response
 message SdkCloudBackupCreateResponse {
+	// TaskId is the id of the task performing the backup
```

Comment:
tabs are wrong. Check out the top of the file for VIM settings.

Reply:
Okay.